CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # bedrock-server ChangeLog
 
+## 3.1.1 - TBD
```

Comment:
So it looks like `3.1.0` hasn't been released yet -- this stuff should move down there. Also, since it adds API surface (even though it's private), it should be a minor release. If we're going to call out specific functions that are being exposed we should explicitly say they are for testing purposes only as well -- just to be extra clear to anyone reading it that doesn't pay attention to the `_` private function convention.